### ISSUE TITLE
Slider lifecycle when update ticks dynamically

### ISSUE THREAD

**Issue by oseasmoran73** [angular-slider/ng5-slider#89]
Hello,

Hope all is well. I been using the slider and it has been great. I did run into an interesting issue where ticks and tick values don't update even with `manualRefresh.emit()`. The only way to get it to update is when you interact with the slider (first click).

I have noticed that `updateTicksScale` function in slider does update the tick properly so that they would be placed correctly, but angular doesn’t detect the new ticks. Adding a `this.changeDetectionRef.detectChanges();` at the end of their function makes should fix this issue. Thoughts?

**Comment by piotrdz** [comment]
You are right, this is a bug and adding the change detection call is a correct way to fix it. I will include the fix in next release.

**Comment by tcoz** [comment]
The 1.2 update that was set to fix the issue I filed and that was linked to this as a duplicate, seems to have an issue. 

I updated our project to 1.2, and the slider became unusable. As soon as you move a slider, the handle "goes bezerk" for lack of a better way to describe it, moving erratically all over the place until finally pinning to the max. Downgrading back to 1.1.14 removed the issue. 

The error is related to the onMove handler:

push../node_modules/ng5-slider/esm5/ng5-slider.js.SliderComponent.onMove

ultimately resulting in this error:

attr.js:31 Error: <circle> attribute r: Expected length, "NaN".

attr.js:31 Error: <circle> attribute r: Expected length, "NaN".

**Comment by piotrdz** [comment]
@tcoz I think you got the wrong issue number - your issue was a duplicate of #80. As I posted in other ticket, please raise a new issue for the problem you are experiencing with reproduction steps.

For this issue #89, I have not done any work yet.

**Comment by piotrdz** [comment]
I have now made a change to fix the original problem in this issue. Along with fix for #105, I believe this will solve problems with updating ticks parameters dynamically.

I have also discovered that there was one bug when using simple slider with ticks, which could result in erratic behaviour as described by @tcoz. This should also be fixed with those changes.

I will release a new version with these changes quite soon.
